src/domains/Invite/InviteForm/InviteForm.tsx

```diff
@@ -0,0 +1,468 @@
+/* eslint-disable react/jsx-one-expression-per-line */
```

Comment:
For the main part of this component, we should probably add a "Download an example file" button to make it easier.  I will go over how to do this in our next meeting.

Also, we should maybe add some "things to check" on the file preview page if the data does not look right.  Such as, "make sure to include headers -- download and look at the example file if you're not sure what this means"